src/m365/teams/commands/conversationmember/conversationmember-add.ts

```diff
@@ -0,0 +1,256 @@
+import * as chalk from 'chalk';
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import commands from '../../commands';
+import { Channel } from '../../Channel';
+import { Team } from '../../Team';
+import Utils from '../../../../Utils';
+import * as os from 'os';
+import GraphCommand from '../../../base/GraphCommand';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId?: string;
+  teamName?: string;
+  channelId?: string;
+  channelName?: string;
+  userId?: string;
+  userDisplayName?: string;
+  owner: boolean;
+}
+
+class TeamsConversationMemberAddCommand extends GraphCommand {
+  public get name(): string {
+    return commands.TEAMS_CONVERSATIONMEMBER_ADD;
+  }
+
+  public get description(): string {
+    return 'Adds or sets a conversationmember in a private channel.';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.teamId = typeof args.options.teamId !== 'undefined';
+    telemetryProps.teamName = typeof args.options.teamName !== 'undefined';
+    telemetryProps.channelId = typeof args.options.channelId !== 'undefined';
+    telemetryProps.channelName = typeof args.options.channelName !== 'undefined';
+    telemetryProps.userId = typeof args.options.userId !== 'undefined';
+    telemetryProps.userDisplayName = typeof args.options.userDisplayName !== 'undefined';
+    telemetryProps.owner = args.options.owner;
+    return telemetryProps;
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    let teamId: string = '';
+    let channelId: string = '';
+
+    this
+      .getTeamId(args)
+      .then((_teamId: string) => {
+        teamId = _teamId;
+        return this.getChannelId(_teamId, args);
+      }).then((_channelId: string) => {
+        channelId = _channelId;
+        return this.getUserId(args);
+      }).then((userIds: string[]) => {
+        const endpoint: string = `${this.resource}/v1.0/teams/${encodeURIComponent(teamId)}/channels/${encodeURIComponent(channelId)}/members`;
+        const roles: string[] = args.options.owner ? ["owner"] : [];
+        const tasks: Promise<void>[] = [];
+        for (const userId of userIds) {
+          tasks.push(this.addUser(userId, endpoint, roles));
+        }
+        return Promise.all(tasks);
+      }).then((): void => {
+        if (this.verbose) {
+          logger.logToStderr(chalk.green('DONE'));
```

Comment:
We're doing this centrally now, so it's no longer needed to do it explicitly in each command.

Reply:
Nice. I'll keep a note of that for future PRs 👍